lib/snippets/kubernetes-deploy

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env ruby
+
+# Usage: kubernetes-deploy <app's namespace> <kube context>
+
+# Prerequisites:
+#  - kubectl 1.5.1+ binary must be available in the shipit machine's path
+#  - ENV['KUBECONFIG'] must point to a valid kubeconfig file that includes all the contexts you want to deploy to
```

Comment:
So if I understand correctly, that kubeconfig is specific to each application deployed? It's the kind of thing that will be in the repo being deployed right?

If so then I think a script argument would be better than an env var. Also if there is any semi-standard path for it, the argument should default to it.